pkg/cdi/cache.go

```diff
@@ -288,6 +303,35 @@ func (c *Cache) WriteSpec(raw *cdi.Spec, name string) error {
 	return spec.Write(true)
 }
 
+// RemoveSpec removes a Spec with the given name from the highest
+// priority Spec directory. This function can be used to remove a
+// Spec previosuly written by WriteSpec() using the same. If the
+// file exists and its removal fails RemoveSpec returns an error.
+func (c *Cache) RemoveSpec(name string) error {
+	var (
+		specDir string
+		path    string
+		err     error
+	)
+
+	specDir, _ = c.highestPrioritySpecDir()
```

Comment:
Does it mean that RemoveSpec could only remove CDI files from the directory with the highest priority?

Reply:
Yes. It is the only/same directory where WriteSpec() writes Spec files, so RemoveSpec() removes only from that same directory.

Reply:
So, not every spec in cache can be removed using RemoveSpec, but only those that are located in the directory with the highest priority?